### PR TITLE
Compilation and indentation fixes

### LINKED ISSUE #18: functions are broken after formatting file
**before:**

	function test()
	    local self = {}

	    self.handle = function()
	        local var = nil
	        --
	        -- long comment ...........................................................................................................................
	        --
	    end

	    return self
	end

**after(The format is even worse):**

	function test()
	    local self = {}

	    self.handle =
	        function()
	        local var = nil
	        --
	        -- long comment ...........................................................................................................................
	        --
	    end

	    return self
	end

### PR DESCRIPTION
**Pull request by ThaisRobba**
Master version was not compiling - wrong type on util.ts.

Fixes indentation issues when multiple assignments are on the right side:

```lua
-- before:
local a,
    b,
    c,
    d,
    e,
    f =
    emptyFunction(),
emptyFunction(
    function()
        return true
    end
)

-- after:
local a,
    b,
    c,
    d,
    e,
    f =
    emptyFunction(),
    emptyFunction(
        function()
            return true
        end
    )
```

Also fixes #18

### CODE REVIEW

File: src/printer.ts [hunk 1]
```diff
@@ -213,9 +213,19 @@ function printNodeNoParens(path: FastPath, options: Options, print: PrintFn) {
                 if (node.init.length) {
                     operator = ' =';
 
-                    right.push(
-                        join(concat([',', line]), path.map(print, 'init'))
-                    );
+                    if (node.init.length > 1) {
```
Comment (ThaisRobba) [inline src/printer.ts:216]: Not really happy with this.

If we don't always indent:
```lua
-- before:
local a, b, c, d, e, f = emptyFunction(function() return true end)

-- after:
local a, b, c, d, e, f =
    emptyFunction(
    function()
        return true
    end
)
```

If we always indent:

```lua
-- before
self.test = function() end

-- after
self.test = function()
        end
```

So it is an ugly hack for now. @trixnz any suggestions on how to solve this? :)

**Comment by trixnz** [comment]
Wow, nice work!

Few things about the tests; `lua-5.3.4-tests` is reserved for a tests distributed with Lua, so there's a high chance your tests would be stomped if ever these were updated. Could you please move them to an appropriate file under `tests/`? Feel free to create a new test category, if required. 

Additionally, when writing new tests, could you please also update the snapshots with `jest --updateSnapshot`.

File: src/printer.ts (hunk 1 above)
Comment (trixnz) [reply src/printer.ts:216]: Hmm. I'm not sure what the most idiomatic way of dealing with this would be. In the first example, I think the `-- after` block more clearly breaks up the noise and shows that the result is going into `a`, but that is entirely subjective. Obviously the second example is the more common way and I'd say the `-- before` version looks better.

Comment (ThaisRobba) [reply src/printer.ts:216]: I'm sorry, my example was not clear. What I meant is that counting the `node.init.length` is a hack that does not cover all edge cases:

```lua
----------------------------
-- function self:test() end

-- [OK] indent if node.init.length > 1
function self:test()
end

-- [OK] always indent
function self:test()
end

----------------------------
-- self.test = function() end

-- [OK] indent if node.init.length > 1
self.test = function()
end

-- [WRONG] always indent
self.test = function()
        end

----------------------------
-- local a, b, c, d, e, f = emptyFunction(function() return true end)

-- [WRONG] indent if node.init.length > 1
local a,
    b,
    c,
    d,
    e,
    f =
    emptyFunction(
    function()
        return true
    end
)

-- [OK] always indent:
local a,
    b,
    c,
    d,
    e,
    f =
    emptyFunction(
        function()
            return true
        end
    )
```

Which is why I'm not quite happy with it.

**Comment by ThaisRobba** [comment]
I'll update the tests asap :)

**Comment by ThaisRobba** [comment]
While updating the tests I noticed that a `lua53` command is spawned. Might be worth adding to the README.md that this command is expected to exist :)

**Comment by ThaisRobba** [comment]
@trixnz Finally updated tests (sorry for the huge delay!).
